client/components/dashboard/dashboard-config-directive.html

```diff
@@ -131,11 +131,37 @@
                  dashboardSvc.DEFAULT_TABLE_PARTITION).label }} (default)"
               ng-model="ngModel.table_partition">
         <md-option ng-repeat="option in dashboardSvc.TABLE_PARTITIONS"
-                ng-value="{{ option.partition }}"
+                ng-value="option.partition"
                 title="{{ option.tooltip }}">{{ option.label }}</md-option>
       </md-select>
     </div>
   </div>
 
+  <div class="pk-sidebar-item">
+    <div class="pk-sidebar-item-label">optimize CURRENT_TIMESTAMP</div>
```

Comment:
Add a tooltip or help link? This is not really self-explanatory. "Optimize for" is ambiguous, clarify that this is a tradeoff of speed vs recentness or that this specifically means that new results within the selected granularity will not be visible even after refreshing.